db/migrate/20180912060726_create_schools.rb

```diff
@@ -0,0 +1,17 @@
+class CreateSchools < ActiveRecord::Migration[5.2]
+
+
```

Comment:
Please fix this file according to previous instructions.

Reply:
as it is a migration file, i wont be able to make changes to it